### PR TITLE
User Notifications Part 2

### PR DESCRIPTION
**Pull request by josephkane**
* notifications for cluster details actions

* should be tested and merged in context with https://github.com/supergiant/control/pull/1300

### CODE REVIEW

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 1]
```diff
@@ -386,10 +386,13 @@ export class ClusterComponent implements AfterViewInit, OnDestroy {
         switchMap(() => this.supergiant.Kubes.get(this.clusterId)),
         catchError((error) => of(error)),
       ).subscribe(
-        k => this.renderMachines(k),
+        k => {
+          this.displaySuccess("Node: " + nodeName, "Deleted!");
```
Comment (taaraora) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: may we need to add space? ```" Deleted!"```

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 2]
```diff
@@ -401,16 +404,15 @@ export class ClusterComponent implements AfterViewInit, OnDestroy {
       .afterClosed()
       .pipe(
         filter(isConfirmed => isConfirmed),
-        switchMap(() => this.supergiant.Kubes.delete(this.clusterId)),
-        catchError((error) => of(error)),
+        switchMap(() => this.supergiant.Kubes.delete(this.clusterId))
       ).subscribe(
         res => {
-          this.success(this.clusterId);
+          this.displaySuccess("Kube: " + this.kube.name, "Deleted!");
```
Comment (taaraora) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: may we need to add space? ```" Deleted!"```

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 3]
```diff
@@ -425,10 +427,16 @@ export class ClusterComponent implements AfterViewInit, OnDestroy {
         // can't mutate table data source, polling erases any optimistic updates, so this happens, sorry...
         switchMap((_) => this.deletingApps.add(releaseName)),
         switchMap(res => this.supergiant.HelmReleases.delete(releaseName, this.clusterId, true)),
-        catchError(err => of(err))
       ).subscribe(
-        res => { this.getReleases(releaseName); },
-        err => { this.deletingApps.delete(releaseName); console.error(err); }
+        res => {
+          this.getReleases(releaseName);
+          this.displaySuccess("App: " + releaseName, "Deleted!")
```
Comment (taaraora) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: may we need to add space? ```" Deleted!"```

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts (hunk 1 above)
Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: @taaraora probably not, because all the args are displayed on their own line

**Pull request merged**